news/2016-11-30-IIDEX-2016.html

```diff
@@ -80,13 +80,13 @@ <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">flo
             </header>
 
             <div class="flc-toc-tocContainer toc"> </div>
-            <div class="floe-content floe-news-item">
+            <article class="floe-content floe-news-item">
                 <h2> News </h2>
 
                 <h3>IIDEX 2016</h3>
-                <p class="floe-date">November 30, 2016</p>
+                <time class="floe-date" datetime="2016-11-30">November 30, 2016</time>
                 <p>
-                    The IDRC participated in the <a href="http://www.iidexcanada.com/conference/summits/national-summit-on-accessibility-2016">IIDEX National Summit on Accessibility</a> on November 30th, 2016.
+                    The IDRC participated in the <a href="https://tbs.iidexcanada.com/conference/seminars2016">IIDEX National Summit on Accessibility</a> on November 30th, 2016.
```

Comment:
Propose to remove this link since it's not showing us anything about the National Summit on Accessibility (which you will eventually find after scrolling way down in [https://tbs.iidexcanada.com/conference/seminars2016#Wednesday])

Reply:
I think it's still useful to find the information, even though they unfortunately no longer direct link. Perhaps there is a way to better represent the link. Any thoughts?

Reply:
In that case, a more descriptive link would be better. Something like:

The IDRC participated in the IIDEX National Summit on Accessibility on November 30th, 2016 (<a href="https://tbs.iidexcanada.com/conference/seminars2016">2016 IIDEX Seminar Schedule</a>).

That way a visitor will know exactly where the link will take them.